howtos/how-to-add-a-create_new_chat-button-in-talkjs/create-new-chat.js

```diff
@@ -0,0 +1,95 @@
+const $targetEl = document.getElementById('crud-modal');
+const modal = new Modal($targetEl)
+const appId = "<Add Your TalkJS APP ID Here>"
+
+Talk.ready.then(function () {
+    var me = new Talk.User({
```

Comment:
It would be preferable to use `const` and `let` rather than `var` throughout

(I notice our getting started docs aren't very clear on this! I'll look to update them, but it would be better to match the code snippets in the  "JS (ESNext)" tab rather than the "JS" tab)